src/main/java/org/myrobotlab/service/KeyboardSim.java

```diff
@@ -0,0 +1,157 @@
+package org.myrobotlab.service;
+
+import java.awt.AWTException;
+import java.awt.Robot;
+import java.awt.event.KeyEvent;
+
+import org.myrobotlab.framework.Service;
+import org.myrobotlab.framework.ServiceType;
+import org.myrobotlab.logging.Level;
+import org.myrobotlab.logging.LoggerFactory;
+import org.myrobotlab.logging.LoggingFactory;
+import org.slf4j.Logger;
+
+/**
+ * @author MaVo (MyRobotLab) / LunDev (GitHub)
+ */
+
+public class KeyboardSim extends Service {
+	static final long serialVersionUID = 1L;
+	static final Logger log = LoggerFactory.getLogger(KeyboardSim.class);
+
+	private Robot robot;
+
+	public KeyboardSim(String n) {
+		super(n);
+	}
+
+	@Override
+	public void startService() {
+		super.startService();
+		try {
+			robot = new Robot();
+		} catch (AWTException e) {
+			log.error("could not create java.awt.Robot");
+			log.error(e.getMessage());
+		}
+	}
+
```

Comment:
I think we starting Peers in startService is preferable because it give the user an option to do a Runtime.create() without the peers being created.  But I can't think of a reason to delay the creation of the awt Robot.  I'd put it in the constructor.

The error/exception catch you can do  on a single line.  The way you currently have it you want get the stack trace. But if you do it this way :
log.error("could not create java.awt.Robot", e);   you do.

So mebbe get rid of  startService completely & constructor like this 
```java
public KeyboardSim(String n) {
		super(n);
                try {
			robot = new Robot();
		} catch (Exception e) {
			log.error("could not create java.awt.Robot", e);
		}
	}
```
Some might disagree too but I'd catch Exception, as any exception coming from new'ing a Robot would be FATAL although maybe only  AWTException  has been documented...